client/gutenberg/extensions/contact-info/email/save.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import textMatchReplace from 'gutenberg/extensions/presets/jetpack/utils/text-match-replace';
+
+const renderEmail = inputText => {
+	return textMatchReplace(
```

Comment:
should we have this as a sanitizeEmail shared inside utils instead?

Reply:
You can just use email validator:

```js
import emailValidator from 'email-validator';
```

See https://github.com/Automattic/wp-calypso/blob/84e7e6165f1d023256218dc1351751b6c947a33d/client/gutenberg/extensions/simple-payments/edit.js#L294-L317

Reply:
This function ins not trying to validate emails but converts emails to links given a string. 
As far as I know `email-validator` is not able to do that. 

I don't think we should use place it in `utils` until we find a use for it somewhere else.